azure-iot-device/azure/iot/device/iothub/pipeline/pipeline_stages_iothub_mqtt.py

```diff
@@ -60,14 +61,17 @@ def _run_op(self, op):
             # The customer user agent string would simply be appended to the end of this username, in URL Encoded format.
             query_param_seq = [
                 ("api-version", pkg_constant.IOTHUB_API_VERSION),
-                ("DeviceClientType", ProductInfo.get_iothub_user_agent()),
+                (
+                    "DeviceClientType",
```

Comment:
>DeviceClientType [](start = 21, length = 16)

so previously user passed product info was not set here

Reply:
Yeah, it was set differently down below. There has been no behavioural change, I just simplified the code a little bit so it does this all at once, and the relationship is clear.

---
In reply to: [399728411](https://github.com/Azure/azure-iot-sdk-python/pull/514#discussion_r399728411) [](ancestors = 399728411)